src/hash/rpo/mod.rs

```diff
@@ -38,7 +38,7 @@ const DIGEST_SIZE: usize = DIGEST_RANGE.end - DIGEST_RANGE.start;
 const NUM_ROUNDS: usize = 7;
 
 /// The number of byte chunks defining a field element when hashing a sequence of bytes
-const BINARY_CHUNK_SIZE: usize = 7;
+const BINARY_CHUNK_SIZE: usize = 8;
```

Comment:
Wouldn't this lead to collisions? (I mentioned this in https://github.com/0xPolygonMiden/crypto/pull/46#issuecomment-1406172233). For example, wouldn't 0 and 2^64 - 2^32 + 1 hash to the same value?

Reply:
Yes, the modulus indeed would cause a collision there. I updated the tests to check for that case + added a collision search for 0..255 zeroes hashed.